src/libraries/System.Net.Http/src/System/Net/Http/HttpClient.CreateDefaultHandler.Android.cs

```diff
@@ -0,0 +1,46 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Reflection;
+using System.Diagnostics.CodeAnalysis;
+
+namespace System.Net.Http
+{
+    public partial class HttpClient
+    {
+        private const string CustomHandlerTypeEnvVar = "XA_HTTP_CLIENT_HANDLER_TYPE";
```

Comment:
It should use the standard .NET AppContext mechanism instead. I'd also suggest to come up with name which can be shared on all configuration which have OS provided handler.

Reply:
Does it mean we're fine with braking the current behavior (and VS/VSfM pluggins)?

Reply:
Not necessary it will have to be handled by a migration tool somehow anyway